### PR TITLE
Issue 33/decode merge name

### LINKED ISSUE #33: Update code to handle different merge_name format
Format 1 (standard):
"2020-06-18T160015_ILWGS_TMGCUC_NWD190874_256256_2-FLOWCELL-H2JV7DSXY-H7NFJDSXY-H7NHMDSXY"

Format 2:
"TMGCUC.NWD438629-1_2AMP-FLOWCELL-HNL7HDSXX-HWHLTDSXX-HMWGFDSXX"

Format 3:
"NWD282009-LIB-ILWGS_TMGCUC_NWD282009_242134_2-ILWGS_TMGCUC_NWD282009_242134_3"
"NWD718492-LIB-ILWGS_TMGCUC_NWD718492_242146_2"

### PR DESCRIPTION
**Pull request by jekwatt**
Fixes #33.

More docstrings for functions will be addressed in #29.

- Add `decode_merge_name` in business.py
- Move `load_merge_report` to business.py
- Use descriptive function name `load_sixty_seventy_coverage_metrics`
- Add docstrings for `load_sixty_seventy_coverage_metrics`
- Remove shebang lines
- Remove `if __name__ == "__main__"` lines
- Minor updates
- Upgrade version to `1.0.1`

### CODE REVIEW

File: pm_utils/business.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+# After a blank line, import third-party libraries
+import pandas as pd
+
+# After another blank line, import local libraries
+from .utils import normalize_name
+from .rpt_columns import rpt_merge_cols
+from .mappings import STUDY_MAPPING
+
+logger = logging.getLogger("business in pm_utils")
```
Comment (itsmarcy) [inline pm_utils/business.py]: Using `__name__` is usually the standard

**Review by itsmarcy — APPROVED** [review]
Looks good to me

File: pm_utils/business.py (hunk 1 above)
Comment (walkerh) [reply pm_utils/business.py]:
```python
logger = logging.getLogger(__main__)
```

Comment (jekwatt) [reply pm_utils/business.py]: Should really use `__name__` instead of `__main__`.
Will fix this in another branch.

**Pull request merged**